lib/vdsm/network/link/setup.py

```diff
@@ -45,7 +45,35 @@ def __init__(self):
     def types_dict(self):
         return self._types_dict
 
-    def convert_value(self, key, value):
+    def parse_nets_bridge_opts(self, opts):
+        """
+        param: opts: vdsm.network.nmstate.bridge_util.NetworkConfig bridge opts
+        """
+        if not opts:
+            return {}
+        opts_str = opts.replace('_', '-')
+        bridge_opts = self._string_to_dict(opts_str, ' ', '=')
+        try:
+            for key, value in bridge_opts.items():
+                bridge_opts[key] = self._convert_value(key, value)
+        except KeyError:
+            raise NmstateBridgeOptionNotSupported(
+                f'{key} is not a valid nmstate bridge option'
+            )
+        return bridge_opts
+
+    @classmethod
+    def parse_bond_options(cls, options):
```

Comment:
The name of the class should change if we want to have bond option there. Also by doing this the class interface is inconsistent, one is classmethod, second is called on instance. It should be unified,

Reply:
done